packages/mjml-cli/src/client.js

```diff
@@ -46,9 +46,10 @@ const render = (bufferPromise, { min, output, stdout, fileName, level }) => {
     .then(result => {
 
       const { html, errors } = result
+      const skipValidation = level === 'skip'
```

Comment:
I don't really understand the reason behind that. Could you explain a bit please ? If level is skip, mjml-core send a empty array.